macros/LearnSidebar.ejs

```diff
@@ -710,6 +710,31 @@ var text = mdn.localStringMap({
       'Tools_and_setup': '工具與設定',
       'Design_and_accessibility': '設計與親合度',
     'How_to_contribute': '如何貢獻'
+  },
+  'fr-FR': {
+    'Complete_beginners': 'Les débutants commence ici!',
```

Comment:
Sorry for this disgracious miastake! I'll change that
Now, I choose "Les débutants commencent ici" because it is how the page is named, I thought it would be more consistent if the title is the same as the page.